app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/yourinfo/YourInfoTab.kt

```diff
@@ -164,8 +171,8 @@ internal fun YourInfoTab(
       Spacer(Modifier.height(8.dp))
     }
     CoverageRows(coverageItems, Modifier.padding(horizontal = 16.dp))
-    if (allowChangeAddress || allowEditCoInsured) {
-      Spacer(Modifier.height(16.dp))
+    Spacer(Modifier.height(16.dp))
+    if (!isTerminated) {
```

Comment:
We still do want to hide this button if both of these features are off.
We do use this check for Demo mode I know for sure.

Reply:
Fixed 👍